api/dfx/hooks/auth.hook.ts

```diff
@@ -23,6 +25,10 @@ export function useAuth(): AuthInterface {
     return await call({ url: AuthUrl.signUp, method: 'POST', data: { address, signature, walletId: 12 } });
   }
 
+  function getOwnershipMessage(address: string): string {
+    return `${message}${address}`;
+  }
```

Comment:
Why don't you just use `getSignMessage`?

Reply:
This is a @TaprootFreak recommendation for making the ownership proof faster. In addition to that, getSignMessage call will fall unused once we implement https://github.com/DFXswiss/btc-wallet/issues/43, so we can avoid that extra api call.

Reply:
Ok, then we can probably remove `getSignMessage` completely as it's doing the exact same thing 😉  the auth endpoint just combines the sign-up/in endpoints, but you will still need the signature, and thus the ownership message, for it.

Reply:
that's exactly the point, the ownership message will live frontend side as cake wallet did [here](https://github.com/cake-tech/cake_wallet/pull/1410/files#diff-e5be7b4c0d3de609ac34e5211935078784bfac6502c1edf6e52fe137e723d37bR76) for issue #43.

I will not change how the signature is generated but from where we get the message to be signed. Currently we get the ownership message calling the backend via `getSignMessage` after this MR we can get it also from `getOwnershipMessage` and deprecated the api call in the next PR.


**_console.log of the return of getSignMessage (omit: ### getSignMessage ### ):**

<img width="572" alt="Screenshot 2024-05-14 at 5 35 20 AM" src="https://github.com/DFXswiss/btc-wallet/assets/48102227/b43cef75-eaef-498a-9eed-a58aa6936185">

Reply:
wait, I think I found that would resolve the thing :)